examples/js/loaders/MMDLoader.js

```diff
@@ -2219,7 +2219,7 @@ THREE.MMDLoader.prototype.createMesh = function ( model, texturePath, onProgress
 
 						};
 
-						function detectTextureTransparency ( image, uvs ) {
+						function detectTextureTransparency ( image, uvs, flipY ) {
```

Comment:
To get pixcel data of image data which corresponds to uv.
If flipY is true, y should be flipped.

https://github.com/mrdoob/three.js/pull/8472/files#diff-d8334f11849f0a2342b64fee1738146fR2296